packages/edit-widgets/src/store/actions.js

```diff
@@ -24,6 +24,7 @@ import {
 	POST_TYPE,
 	WIDGET_AREA_ENTITY_TYPE,
 } from './utils';
+import { store as editWidgetsStore } from '../store';
```

Comment:
Didn't we create a file that exports the constant with the name of the store to avoid cyclic dependencies between files in other packages for the store implementation?

Reply:
My bad, that's true. Fixing it.

Reply:
@gziolo done!

Reply:
I think you updated too much. I meant only all files inside `edit-widgets/src/store/*`. It's perfectly fine to reference `editWidgetsStore` imported from `edit-wdigets/src/store/index/js` from files in `edit-widgets/src/components/*` and the rest 😄

Reply:
Oh, I see... I thought that referencing the store inside its own package could also cause cyclic dependencies. I'll fix it tomorrow. Sorry for the confusion 😅...

Reply:
No worries. I really appreciate all your work done some far, you rock 🥇